clients/js/main.ts

```diff
@@ -254,11 +257,18 @@ yargs(hideBin(process.argv))
           describe: "Module to query",
           type: "string",
           choices: ["Core", "NFTBridge", "TokenBridge"],
+        })
+        .option("emitter", {
+          alias: "e",
+          describe: "Print in emitter address format",
+          type: "boolean",
+          default: false,
+          required: false,
         });
     },
     async (argv) => {
       assertChain(argv["chain"]);
-      assertEVMChain(argv["chain"]);
+      // assertEVMChain(argv["chain"]);
```

Comment:
Do you want to just remove this line or comment why we would use it in the future?

Reply:
I will remove it.